pythonforandroid/bootstraps/service_only/__init__.py

```diff
@@ -9,7 +9,9 @@ class ServiceOnlyBootstrap(Bootstrap):
 
     name = 'service_only'
 
-    recipe_depends = ['genericndkbuild', ('python2', 'python3', 'python3crystax')]
+    recipe_depends = list(
+        set(Bootstrap.recipe_depends).union({'genericndkbuild'})
```

Comment:
wait that's not the same right?
```python
>>> list(set(['python2', 'python3', 'python3crystax']).union({'genericndkbuild'}))
['python2', 'python3crystax', 'python3', 'genericndkbuild']
```
Before we had the following result:
```python
['genericndkbuild', ('python2', 'python3', 'python3crystax')]
```
See how it was a two dimensional list I guess it was on purpose. Did you purposely flatten it? Won't it break some features then?

Reply:
@AndreMiras it does the same thing as before, this is what it actually compares to `list(set([('python2', 'python3', 'python3crystax')]).union({'genericndkbuild'}))`. It's not flattened

Reply:
Try to treat `Bootstrap.recipe_depends` as a black box. It can be a full regular flat list, it can contain tuple members as before. It's just whatever previous dependencies were around. As for `('python2', 'python3', ...)` that will be a tuple entry inside that list as before, and remain one during conversion to a set. There's really not much of a difference

Reply:
did that convince you, or do you still need anything else?

Reply:
OK cool so basically `Bootstrap.recipe_depends` is already a two dimension list? That's great then, I was assuming `Bootstrap.recipe_depends` was simply a list
Thanks :+1: